terraform/workspace_variables/sandbox.tfvars

```diff
@@ -18,3 +18,8 @@ statuscake_alerts = {
     node_locations = ["UKINT", "UK1", "MAN1", "MAN5", "DUB2"]
   }
 }
+
+key_vault_name           = "s121p01-shared-kv-01"
+key_vault_resource_group = "s121p01-shared-rg"
+key_vault_app_secret_name      = "PUBLISH-APP-SECRETS-PRODUCTION"
```

Comment:
The sandbox secrets may be different than the prod ones. Shall we create a separate sandbox one?

Reply:
yes, have already created `PUBLISH-APP-SECRETS-SANDBOX` & `BAT-INFRA-SECRETS-SANDBOX`